docs/guides/references/_changelogs/12.4.0.md

```diff
@@ -0,0 +1,50 @@
+## 12.4.0
+
+_Released 1/24/2023_
+
+**Features:**
+
+- Added new
+  [`experimentalSkipDomainInjection`](/guides/references/experiments#Experimental-Skip-Domain-Injection)
+  flag to disable Cypress from setting `document.domain` on injection. Addresses
+  [#2367](https://github.com/cypress-io/cypress/issues/2367),
+  [#23958](https://github.com/cypress-io/cypress/issues/23958),
+  [#24290](https://github.com/cypress-io/cypress/issues/24290), and
+  [#24418](https://github.com/cypress-io/cypress/issues/24418).
+- Added new
+  [`experimentalMemoryManagement`](/guides/references/experiments#Configuration)
+configuration option to improve memory management in Chromium-based browsers. Enable this option with `experimentalMemoryManagement=true` if you have experienced "Out of Memory" issues. Addresses
+  [#23391](https://github.com/cypress-io/cypress/issues/23391).
+- The [`.as`](/api/commands/as) command now accepts an options argument, allowing an alias
```

Comment:
Where is the documentation for this? This should be updated on the 'as' api docs.

Reply:
It was merged into `main` this morning I believe https://github.com/cypress-io/cypress-documentation/commit/59cc496692e10cfa9bd64b70017d342147793b71

Reply:
Aye, whoops, this is my mistake - it went directly into `main` and not `12.4.0`.